index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
 
-    <title>Catalogue</title>
+    <title>Data Tools</title>
```

Comment:
😮

Reply:
Yea, just changing the default so that it's hopefully not too confusing for anyone :)